<?xml version="1.0"?>
<robot name="imu" xmlns:xacro="http://www.ros.org/wiki/xacro">

  <xacro:arg name="prefix" default="" />
  <xacro:arg name="mode" default="simulation_gazebo" />
  <xacro:arg name="name" default="imu" />
  <xacro:arg name="rate" default="100" />
  <xacro:arg name="parent_link" default="base_link" />
  <xacro:arg name="xyz" default="0.0 0.0 0.0" />
  <xacro:arg name="rpy" default="0.0 0.0 0.0" />
  <xacro:arg name="mesh_visual" default="false" />
  <xacro:arg name="ros_namespace" default="/" />

  <xacro:include filename="$(find romea_imu_description)/urdf/imu.xacro" />
  <xacro:imu_sensor
    prefix="$(arg prefix)"
    mode="$(arg mode)"
    name="$(arg name)"
    type="gladiator"
    model="landmark40_ahrs"
    rate="100"
    xyz="$(arg xyz)"
    rpy="$(arg rpy)"
    parent_link="$(arg parent_link)"
    mesh_visual="$(arg mesh_visual)"
    ros_namespace="$(arg ros_namespace)" />

</robot>